assets/sass/utilities/_margin.scss

```diff
@@ -18,6 +18,10 @@
 
 .googlesitekit-plugin {
 
+	.googlesitekit-margin-right-1 {
+		margin-right: 0.5rem;
+	}
```

Comment:
This is a little tricky as we haven't really established a scale to use for utilities yet. We also have different units in use in different places and most of our margin styles appear to be using `px` currently.

I don't think we actually need this class though, we just need to add an `mdc-button__icon` to the icon and the spacing is handled for us 👍